microsoft-iis-mixin/config.libsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  _config+:: {
+    dashboardTags: ['iis-mixin'],
```

Comment:
could change to `microsoft-iis-mixin` to be more consistent (and change tags in dashboards appropriately)